saleor/graphql/product/tests/test_product.py

```diff
@@ -1833,15 +1833,11 @@ def test_filter_products_by_wrong_attributes(user_api_client, product, channel_U
 
 
 def test_sort_products(user_api_client, product, channel_USD):
-    # set price and update date of the first product
-    product.minimal_variant_price_amount = 10
     product.updated_at = datetime.utcnow()
     product.save()
 
-    # Create the second product with higher price and date
     product.pk = None
     product.slug = "second-product"
-    product.minimal_variant_amount = 20
```

Comment:
Shouldn't this test check sorting by the discounted price? I think that's how the price sorter works.

Reply:
`PRICE` is sorting by price amount in ProductVarianChannelListing, we have other sorter to sort by discounted price - `MINIMAL_PRICE`

Reply:
OK I wasn't sure about this, thanks.